### PR TITLE
check string is symmetric or not

### PR DESCRIPTION
**Pull request by ngomy2601**

### CODE REVIEW

**Comment by ngomy2601** [comment]
Updated based on mentor's comments @voquanghoa

File: src/main/java/com/tieumy/exercise/StringEx02.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    public String reverseString(final String input) {
+        final StringBuilder str = new StringBuilder();
+        for (int i = input.length() - 1; i >= 0; i--) {
+            str.append(input.charAt(i));
+        }
+        return str.toString();
+    }
+
+    public boolean IsSymmertrical(final String input) {
```
Comment (nguyenvanman) [inline src/main/java/com/tieumy/exercise/StringEx02.java]:
```suggestion
    public boolean isSymmertrical(final String input) {
```

File: src/main/java/com/tieumy/exercise/StringEx02.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+        String reversedString = reverseString(input);
```
Comment (nguyenvanman) [inline src/main/java/com/tieumy/exercise/StringEx02.java]: Tìm cách không dùng `reverseString`

**Review by nguyenvanman — CHANGES_REQUESTED** [review]

File: src/test/java/com/tieumy/exercise/StringEx02Test.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    @Test
+    void isSymmertrical() {
+        final StringEx02 stringEx02 = new StringEx02();
+        String validInput = "abcdcba";
+        String invalidInput = "abcde";
+        boolean valueTrue = true;
+        boolean valueFalse = false;
+        assertEquals(valueTrue, stringEx02.IsSymmertrical(validInput));
```
Comment (voquanghoa) [inline src/test/java/com/tieumy/exercise/StringEx02Test.java]: Dùng assertTrue/assertFalse

File: src/main/java/com/tieumy/exercise/StringEx02.java (hunk 2 above)
Comment (ngomy2601) [reply src/main/java/com/tieumy/exercise/StringEx02.java]: DONE

File: src/test/java/com/tieumy/exercise/StringEx02Test.java (hunk 3 above)
Comment (ngomy2601) [reply src/test/java/com/tieumy/exercise/StringEx02Test.java]: DONE

**Pull request merged**